.github/main.workflow

```diff
@@ -4,13 +4,18 @@ workflow "Verify the module" {
 }
 
 action "install" {
-  uses = "actions/npm@1.0.0"
+  uses = "actions/npm@master"
   args = "ci"
 }
 
+action "peer_install" {
+  uses = "actions/npm@master"
+  args = "install --no-save @google-cloud/dlp"
```

Comment:
actually i don't think you have to change the workflow file at all to get peer dependencies to work. See my next comment.

Reply:
> as for the checks not running: This seems to be an unfortunate limitation of github actions as of right now. Seems like the creator of the PR (@tbjers) has to be also part of the beta in order for the actions to run (see https://github.community/t5/GitHub-API-Development-and/Run-a-GitHub-action-on-pull-request-for-PR-opened-from-a-forked/td-p/15426).
> I'll find a way to run them myself or maybe manually once the MR is in good shape if you can't get into the actions beta until then.

@geekflyer I think I am part of the beta now. We'll see what happens when I push my changes.